src/components/Covid19.jsx

```diff
@@ -16,16 +16,17 @@ import LocationFilter from './LocationFilter'
 import FilterSection from './FilterSection'
 import StickyActionButton from './StickyActionButton'
 
-import { loadData } from '../utils/loadData'
 import useMobile from '../hooks/useMobile'
 import useLADs from '../hooks/useLADs'
 import useLineages from '../hooks/useLineages'
 import useLALookupTable from '../hooks/useLALookupTable'
 import useDates from '../hooks/useDates'
 
-const data = loadData()
+const Covid19 = ({ lineColor = 'blueGray', tiles, data }) => {
+  if (data.colors === undefined) {
```

Comment:
would suggest to move this to DataProvider and potentially put it in `useMemo` 👍🏽

Reply:
Great point - ended up putting this in the fetch logic that only gets called once